etna/whatson/models.py

```diff
@@ -837,23 +840,32 @@ class ExhibitionPage(
         help_text=_("The location of the exhibition within the venue."),
     )
 
-    location_link_text = models.CharField(
-        verbose_name=_("location link text"),
+    location_address = RichTextField(
+        verbose_name=_("location address"),
         null=True,
-        help_text=_("The text for the location section."),
+        blank=True,
+        help_text=_("Leave blank to default to TNA address."),
```

Comment:
Have we sought advice on these labels from content design? This would be my feeling:

```suggestion
        help_text=_("Leave blank to show TNA address."),
```

Reply:
I copied the help text from Ashley's doc so it should all be good with them 😃